pkg/cmd/experimental/syncgroups/ad/ldapinterface.go

```diff
@@ -22,18 +24,15 @@ func NewADLDAPInterface(clientConfig *ldaputil.LDAPClientConfig,
 		userQuery:                 userQuery,
 		userNameAttributes:        userNameAttributes,
 		groupMembershipAttributes: groupMembershipAttributes,
-		ldapGroupToLDAPMembers:    map[string][]*ldap.Entry{},
+		cachedGroupedUsers:        map[string][]*ldap.Entry{},
```

Comment:
This name change is bad.  We're dealing with two different things called groups, you must qualify.  If you're matching code somewhere else, fix it there too.